symplyphysics/laws/thermodynamics/chemical_potential_is_derivative_of_thermodynamic_potential.py

```diff
@@ -0,0 +1,69 @@
+from sympy import Eq, Symbol as SymSymbol, Derivative
+from symplyphysics import (
+    units,
+    dimensionless,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Chemical potential can be described as the amount of energy the system absorbs or releases
+## due to the change of particle number. Thermodynamically it can be found by differentiating
+## thermodynamic potentials with respect to the particle number keeping the other two natural
+## variables of the potential constant.
+
+# Law: mu = (dX/dN)_(a, b)
+## mu - chemical potential
+## X - thermodynamic potential
+## N - particle count
+## a, b - other two natural variables of X
+
+# Note
+## - mu = (dU/dN)_(S, V) = (dF/dN)_(T, V) = (dH/dN)_(S, p) = (dG/dN)_(T, p) = (dΩ/dN)_(T, V)
```

Comment:
This general law looks quite vague. Might be a good idea to write all special cases of this law, as you described here.

Reply:
I wanted to make it general enough to encompass all cases, but it's true that now it's vague